### ISSUE TITLE
Replicating Cost-Performance curves (Figure 6)

### ISSUE THREAD

**Issue by tchittesh** [lxasqjc/Deformation-Segmentation#11]
Could you please share more info on how to replicate the cost-performance curves from Figure 6? Are the training hyperparameters kept the same as in the quick_start_bash scripts, or is the num_epochs for example and learning rate tuned per experiment? At what lower resolutions are the experiments run to generate the curves?

**Comment by lxasqjc** [comment]
Hi @tchittesh, the num_epochs and learning rate are the same, but we tuned the hyperparameters as in Table 2-5 of section "A.1 SENSITIVITY ANALYSIS ON HYPERPARAMETERS" in the APPENDIX.
